Jenkinsfile

```diff
@@ -29,9 +29,35 @@ pipeline {
             name 'PKR_VAR_cloud'
             values 'aws', 'azure'
           }
+          axis {
+            name 'PKR_VAR_location'
+            values 'us-east-2', 'East US 2', 'East US'
```

Comment:
this is a bit confusing to me with all the excludes =/

Reply:
yes, I agree :'( I've added comment for each as a "best effort". 

We are limited by the current syntax. Evolutions could be:

* Split in 2 matrix (1 for AWS, 1 for Azure) and Build sequentially. But slow :'(
* Split in 2 matrix (1 for AWS, 1 for Azure) when declarative syntax would allow parallel stages with a matrix within
* Switch to scripted syntax, but I'm not willing to spend time on this unless really mandatory. My brain is not wired for scripted pipelines.

Maybe others solutions?

Reply:
Would you have a better solution? I don't see an easy solution to support different Cloud,Region,Platform

Reply:
adding a comment here as well? appropriate location will be picked per cloud provider?

but no i don't have a better solution

Reply:
@timja LGTM, let me add a comment before merging